lib/libbackscrub.h

```diff
@@ -38,4 +38,6 @@ extern void bs_maskgen_delete(void *context);
 // Process a video frame into a mask
 extern bool bs_maskgen_process(void *context, cv::Mat& frame, cv::Mat &mask);
 
+extern cv::Rect calcCropping(int inWidth, int inHeight, int targetWidth, int targetHight);
```

Comment:
As per the code comment - please stick to existing naming convention of `bs_xxx` :)

Reply:
Okay

Reply:
done not pushed